editor/plugins/spatial_editor_plugin.cpp

```diff
@@ -3566,7 +3566,8 @@ SpatialEditorViewport::SpatialEditorViewport(SpatialEditor *p_spatial_editor, Ed
 	ED_SHORTCUT("spatial_editor/freelook_speed_modifier", TTR("Freelook Speed Modifier"), KEY_SHIFT);
 
 	preview_camera = memnew(CheckBox);
-	preview_camera->set_position(Point2(10, 38) * EDSCALE); // Below the 'view_menu' MenuButton.
+	const int &font_size = EditorSettings::get_singleton()->get("interface/editor/main_font_size");
```

Comment:
The fix is fine but I don't see the point of using `const int &` here. It just means the code less readable without any gain, this is a constructor that runs only once, and `EditorSettings::get` returns a non-const Variant anyway so memory will be allocated to store this `int`, so it's overkill to make this a `const int &` where `int` would be just fine.

Reply:
Fixed with https://github.com/godotengine/godot/pull/28827/commits/0f06b09fe6bdaa9b7442877d4cc67a8c80e1db04